builtin_interfaces/Time stamp

float64[3] position
float64[3] position_setpoint

float64[3] velocity
float64[3] velocity_setpoint

float64[3] acceleration
float64[3] acceleration_setpoint

float64 yaw_angle_setpoint

float64[3] desired_acceleration
float64[3] desired_attitude
float64 desired_thrust
